src/styles/blocks/stars.css

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+  position: relative;
+}
+
+.stars__stars-0,
+.stars__stars-1,
+.stars__stars-2,
+.stars__stars-3,
+.stars__stars-4,
+.stars__stars-5 {
+  display: none;
+}
```

Comment:
you should reuse stars from the previous task

Reply:
https://github.com/mate-academy/layout_stars/pull/1745